### PR TITLE
Resolve: Prepare v0.3.14

### LINKED ISSUE #1723: Prepare 0.3.14
## Description

Prepare next release {version}

## TODO

1. Create branch `release/{version-number}` from latest `develop`
1. Bump version number in `package.json`
1. Update CHANGELOG.md
1. Create DRAFT release page at GH linked to `release/{version-number}` branch
1. ADD release notes to DRAFT release page
1. Sign binaries as described [here](https://github.com/thorchain/Resources/blob/master/admin-GPG-keys.md) and add signed message to DRAFT release page
1. Check binaries by running on `macOS`, `Windows` and `Linux`
1. Publish release

### PR DESCRIPTION
**Pull request by veado**
DON'T merge - we need to cherry pick 7271f3f into `develop` only

Closes #1723

### CODE REVIEW

**Review by gromxyz — DISMISSED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
+## Add
+
+- [Mimir] Support `HALT{chain}TRADING` flags for `BTC`, `LTC`, `BCH`, `BNB` [#1722](https://github.com/thorchain/asgardex-electron/issues/1722)
+- [Mimir] Support `PAUSELP{chain}` flags for `BNB`, `BTC`, `BCH`, `ETH`, `LTC` [#1724](https://github.com/thorchain/asgardex-electron/issues/1724)
+
+## Update
+
+- Updated french translation (01-09-2021) [#1717](https://github.com/thorchain/asgardex-electron/pull/1717)
+
+## Fix
```
Comment (gromxyz) [inline CHANGELOG.md:12]: Maybe add #1732 to the fixes too

**Review by gromxyz — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (veado) [reply CHANGELOG.md:12]: It's part of #1724

**Comment by veado** [comment]
7271f3f + 91055f1 have been squashed + merged into `develop` branch

**Pull request closed without merging**